superset-frontend/src/components/ListView/ListView.tsx

```diff
@@ -198,7 +198,7 @@ const ListView: FunctionComponent<Props> = ({
                     {bulkActions.map(action => (
                       // @ts-ignore
                       <MenuItem
-                        key={action.key}
+                        key={action.key || action.name}
```

Comment:
when will an action not have a key?

Reply:
good point. I got a warning thrown in tests (since it's a js file), so I added a fallback. If it were a ts file it would have complained. I'll remove an fix in tests

Reply:
I added this after getting a react error in tests. I've updated the BulkAction type and made key a required field. Also fixed the test